src/Captioning/File.php

```diff
@@ -229,16 +229,19 @@ public function getCueFromStart($_start)
     /**
      * Add a cue
      *
-     * @param mixed $_mixed An cue instance or a string representing the text
+     * @param mixed  $_mixed An cue instance or a string representing the text
      * @param string $_start A timecode
-     * @param string $_stop A timecode
+     * @param string $_stop  A timecode
+     *
+     * @return $this
+     * @throws \Exception
      */
     public function addCue($_mixed, $_start = null, $_stop = null)
     {
         $fileFormat = self::getFormat($this);
 
         // if $_mixed is a Cue
-        if (is_subclass_of($_mixed, __NAMESPACE__.'\Cue')) {
+        if (is_object($_mixed) && class_exists(get_class($_mixed)) && class_exists(__NAMESPACE__.'\Cue') && is_subclass_of($_mixed, __NAMESPACE__.'\Cue')) {
```

Comment:
It's been too long and I don't remember exactly what prompted those checks in particular, but the commit message for that change says "Add additional checks to File::addCue to prevent is_subclass_of warnings". So that line was generating warning messages, which might have messed things up in my dev environment with error reporting enabled. Looking back, I should have added the warning message.